src/sdkToEventsApiConverter.ts

```diff
@@ -59,7 +59,11 @@ export function convertEvents(
     const upload: EventsApi.Batch = {
         source_request_id: mpInstance._Helpers.generateUniqueId(),
         mpid,
-        timestamp_unixtime_ms: new Date().getTime(),
+        timestamp_unixtime_ms: mpInstance._Store.batchTimestampUnixtimeMsOverride
```

Comment:
As much as I love ternary operations, but sometimes they can be hard to read, especially when they're nested and have long variable names.

I'd be fine with you keeping it if you destructure the members or just move it above the `upload` declaration and make it a traditional if/else statement.

Reply:
Cool, sounds good.

Also, I want to confirm one thing: I'm just checking for a truthy value on `batchTimestampUnixtimeMsOverride`, so callers want be able to send `batchTimestampUnixtimeMsOverride: 0`. This is fine for me (as this would likely be a mistake), but just wanted to call it out.